src/elements/notifications/NotificationContent.tsx

```diff
@@ -111,7 +111,7 @@ export const NotificationContent = ({
             {dayjs.unix(timestamp).fromNow(true)}
           </span>
         </div>
-        <button onClick={() => onRemove(id)}>
+        <button className="-m-8 p-8" onClick={() => onRemove(id)}>
```

Comment:
It is best to change the width of the button (the icon will follow) instead of extending the clickable area
also lets make it bigger `w-12` should be fine

Reply:
Thanks for your comment @Ran73Cohen 

How about that ?

![w-12-h-12](https://user-images.githubusercontent.com/6317798/129527168-5bbfc3c3-0cff-416e-9180-a35b3860c78b.png)

Reply:
Great!